main-project/src/boards/dto/create-board.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsDate, IsNotEmpty, IsNumber, IsString } from 'class-validator';
+export class CreateBoardDto {
+  @IsNotEmpty()
+  title: string;
+
+  @IsNotEmpty()
+  description: string;
+
+  @IsNumber()
+  done: boolean;
```

Comment:
```suggestion
@IsNotEmpty()
@IsBoolean()
isDone: boolean;
```

데이터 타입과 유효성 검사 타입을 맞춰줘야 합니다. 그리고 done이 불린 타입이어서 유추해본 결과 0 혹은 1의 값으로 판단되는데 그러면 isDone으로 식별자의 이름을 지어주면 좋습니다 ㅎㅎ

Reply:
오 변수명 정하는 게 어려웠는데 감사합니다 :)